Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -841,7 +841,7 @@ static void registerLogHook()
 #ifdef NDEBUG
         // Don't send messages with types we don't want to log in release. Even though OS_LOG_TYPE_DEFAULT is the minimum,
         // the hook will be called for other subsystems with debug and info types.
-        if (type & (OS_LOG_TYPE_DEBUG | OS_LOG_TYPE_INFO))
+        if (type == OS_LOG_TYPE_DEBUG)
```

Comment:
Seems like a step in the wrong direction. The network process is already getting quarantined all the time. Maybe let's fix that before we start logging even more from the network process?

I think we should just stop forwarding to the Network process until we find a solution to the quarantine issue.

Reply:
I see, although this may not be leading to more logging compared to earlier builds, since this is a partial revert of a very recent change. I think we should be using the == operator here.

Thanks for reviewing, Chris!

Reply:
Why do we forward to the network process and not the UI process? (This whole logging situation is odd to me. As long as debug keeps working in debug builds, I'm happy.)